study.md

```diff
@@ -50,7 +50,13 @@ A kitchen has many items.
     rack, and cinnamon, nutmeg, and vanilla on another shelf.
 
 ```json
-<!-- your answer here -->
+<{
+  "kitchen": {
+    "cupboard": ["bowls", "plates"],
+    "heatingElements": ["small", "medium", "large", "extra large"],
+    "herbsAndSpices": [["chives", "lemon grass", "mint"],["cinnamon", "nutmeg", "vanilla"]],
```

Comment:
You could also represent this with two keys, like `shelfOne` and `shelfTwo`, but this way works too